common/supplemental/units.xml

```diff
@@ -342,6 +342,8 @@ For terms of use, see http://www.unicode.org/copyright.html
        
         <!-- speed -->
         <convertUnit source='knot' baseUnit='meter-per-second' factor='1852/3600' systems="ussystem uksystem si_acceptable"/>
+        <convertUnit source='beaufort' baseUnit='meter-per-second' special='beaufort' systems="metric_adjacent"/>
```

Comment:
except for the hardcoded table, the code is general enough to be applied to any scale of this type. Maybe this could be something like

```
special='scale[0.0, 0.3, 3.4, ..., 55.8]'
```

Reply:
So in the latest version I made the code more generic: The functions converting between scale values and base value take an array of the minima for each scale value (plus an extra at the end to produce a reasonable "midrange" for the highest scale value). The special attribute still takes a string to select which array gets passed into these functions, instead of taking the array itself.